seed/csharp-sdk/alias-extends/src/SeedAliasExtends/SeedAliasExtendsClient.cs

```diff
@@ -22,7 +23,8 @@ public SeedAliasExtendsClient(ClientOptions? clientOptions = null)
 
     public async Task ExtendedInlineRequestBodyAsync(
         InlinedChildRequest request,
-        RequestOptions? options = null
+        RequestOptions? options = null,
+        CancellationToken cancellationToken = default
```

Comment:
we could've just the same had `= null` here right? (checking understanding, not requesting a change)

Reply:
Not in this case - it's idiomatic for the `CancellationToken` to be a required parameter, so following the same style actually makes it a lot easier to propagate the token at every call-site.

For example, consider the HTTP client call where this is used:

```csharp
var response = await httpClient.SendAsync(httpRequest, cancellationToken);
```

You'll actually notice that the signature similarly expects a required value like so:

```csharp
public override Task<HttpResponseMessage> SendAsync(
  HttpRequestMessage request,
  CancellationToken cancellationToken);
```

If we typed this as optional, we'd have to include a ternary at every call-site. For some extra justification, [here](https://stackoverflow.com/questions/22359706/default-parameter-for-cancellationtoken) is a relevant discussion.